Resources/doc/reference/configuration/proxy-client.rst

```diff
@@ -80,9 +80,15 @@ nginx
                 servers: 123.123.123.1:6060, 123.123.123.2
                 base_url: yourwebsite.com
                 purge_location: /purge
+        tags:
+            enabled:        false
 
 For ``servers`` and ``base_url``, see above.
 
+.. warning::
+
+    By now, tagging only works with Varnish, so ``tags`` has to be explicitly disabled.
```

Comment:
#222 will fix that bug. it will however collide with the fixes you are doing in this PR. i propose we check if #222 is green and if so merge that, then i will merge this PR manually and solve the conflict.